options/locale/locale_en-US.ini

```diff
@@ -620,6 +620,17 @@ gpg_token_code = echo "%s" | gpg -a --default-key %s --detach-sig
 gpg_token_signature = Armored GPG signature
 key_signature_gpg_placeholder = Begins with '-----BEGIN PGP SIGNATURE-----'
 verify_gpg_key_success = The GPG key '%s' has been verified.
+ssh_key_verified=Verified Key
+ssh_key_verified_long=Key has been verified with a token and can be used to verify commits matching any activated email addresses for this user.
+ssh_key_verify=Verify
+ssh_invalid_token_signature = The provided SSH key, signature and token do not match or token is out-of-date.
+ssh_token_required = You must provide a signature for the below token
+ssh_token = Token
+ssh_token_help = You can generate a signature using:
+ssh_token_code = echo -n "%s" |ssh-keygen -Y sign -n gitea -f /path_to_your_pubkey
+gpg_token_signature = Armored SSH signature
+key_signature_ssh_placeholder = Begins with '-----BEGIN SSH SIGNATURE-----'
+verify_ssh_key_success = The SSH key '%s' has been verified.
```

Comment:
"The SSH key 1213"
"SSH key 1213"

I think it sounds better without the `The`, as it reference to a specific SSH Key, but after this word the specificity(which key) is given.

Reply:
I can change it, just copied it from the `verify_gpg_key_success` :)

Reply:
Ah I see, let's change it(and also change `verify_gpg_key_success`).